### DISCUSSION TITLE
Choosing a biome for outpost PVP does not work.

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15348] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/1b5888c8-7a34-45cb-9f61-2877c68b15ec)

`  
  <ColdCavernsOutpostPvP
    identifier="coldcavernsoutpost_pvp"
    biomes="any"
    commonness="1"
    Type="Outpost"
    IsPvPLevel="True"
 ...............................................
  </ColdCavernsOutpostPvP>`
  
  If you choose a biome using this, during outpost pvp, it only ever does coldcavernsoutpost_pvp
  
it would be better if you could choose a biome, and then *each* biome had its own outpost_pvp levelgenerationparameter. selecting europanridge would do the same exact thing as selecting coldcaverns, but change the sprites.

![image](https://github.com/user-attachments/assets/1469070d-8227-425b-a01c-65bb57f0078f)
![image](https://github.com/user-attachments/assets/38dad23f-7036-4e41-8a5b-3f8b7418a272)
![image](https://github.com/user-attachments/assets/0aae2038-6509-4afa-8e75-671f1af2654c)
as you can see, this is clearly not europan ridge. 

### Reproduction steps

1. choose a biome in outpost deathmatch
2. watch helplessly as it does not generate a level that appears to be that biome
3. cry

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in the next unstable build.
